checks/apps/cp2k/cp2k_check.py

```diff
@@ -3,149 +3,208 @@
 #
 # SPDX-License-Identifier: BSD-3-Clause
 
+import os
+import shutil
+
 import reframe as rfm
 import reframe.utility.sanity as sn
+import reframe.utility.udeps as udeps
+import uenv
+
+cp2k_references = {
+    "pbe": {"gh200": {"time_run": (65, None, 5, "s")}},
```

Comment:
Do you expect an upper limit of 65*5 or reference +5% ?
```suggestion
    "pbe": {"gh200": {"time_run": (65, None, 5/100, "s")}},
```